### DISCUSSION TITLE
2FA - Brazil

### DISCUSSION THREAD

**Discussion by GugaS1lva** [community/community#52533] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Guys, I have a lot of doubts about the new GitHub 2FA process. I already perform a second verification of my account using the GitHub App on my mobile device. However, every day I still receive a message warning me to enable two-factor authentication. I would like to be able to use my own mobile number, but Brazil is not available in the list of countries. Can someone help me? 😕

**Comment by airtower-luna** [comment, marked as answer]
You can use TOTP (the "authenticator app" thing) instead of SMS, it's actually safer. Just make sure to keep your recovery codes in a safe place and make a backup of the authenticator you use.
